### PR TITLE
favorite page

### PR DESCRIPTION
**Pull request by fturello**

### CODE REVIEW

**Comment by fturello** [comment]
@lchastanet

File: frontend/src/pages/Favorites.jsx [hunk 1]
```diff
@@ -1,11 +1,48 @@
...
 import styles from "../styles/Favorites.module.scss";
 
 export default function Favorites() {
+  const [favorites, setFavorites] = useState([]);
+  useEffect(() => {
+    axios
+      .get(`${import.meta.env.VITE_BACKEND_URL}/video`)
+      .then((res) => res.data)
+      .then((data) => {
+        // console.log(data);
```
Comment (lchastanet) [inline frontend/src/pages/Favorites.jsx]: Debug here

File: frontend/src/pages/Favorites.jsx [hunk 2]
```diff
@@ -1,11 +1,48 @@
...
+              src={favorite.thumbnail}
+              alt={favorite.description}
+            />
+            <div className={styles["title-container"]}>
+              <p>
+                {favorite.title}
+                <div className={styles.trashbin}>
+                  <TrashBinOutline
+                    color="#00000"
+                    // title={}
```
Comment (lchastanet) [inline frontend/src/pages/Favorites.jsx]: commented code here

File: frontend/src/styles/Favorites.module.scss [hunk 3]
```diff
@@ -10,4 +10,33 @@
...
     justify-content: center;
     margin-top: 1rem;
+}
+
+.favorites {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    // flex-wrap: wrap;
```
Comment (lchastanet) [inline frontend/src/styles/Favorites.module.scss]: here again

File: frontend/src/styles/Favorites.module.scss [hunk 4]
```diff
@@ -10,4 +10,33 @@
...
+}
+
+.title-container {
+  width: 80%;
+  flex-direction: row;
+}
+
+.trashbin {
+  display: flex;
+  // margin-top: 0.5rem;
```
Comment (lchastanet) [inline frontend/src/styles/Favorites.module.scss]: + 1

**Review by lchastanet — CHANGES_REQUESTED** [review]

**Review by antoine-droid — APPROVED** [review]
Good 🌵

**Review by antoine-droid — APPROVED** [review]
Good 🌴

**Review by lchastanet — APPROVED** [review]

**Comment by fturello** [comment]
@lchastanet

**Pull request merged**